scripts/Dockerfile

```diff
@@ -13,16 +18,45 @@ RUN add-apt-repository ppa:rock-core/qt4 -y && \
 RUN rm -rf /catkin_ws && \
     mkdir -p /catkin_ws/src && \
     cd /catkin_ws && \
-    catkin init
+    catkin init && \
+    cd /catkin_ws/src && \
+    git clone https://github.com/RUFFY-369/CrazyS.git && \
+    git clone https://github.com/RUFFY-369/mav_comm.git && \
+    cd /catkin_ws
 COPY .rosinstall /tmp/
 RUN vcs import /catkin_ws/src  < /tmp/.rosinstall
 
+
+#CrazyS
+RUN sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+RUN apt install -y curl
+RUN curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
+RUN apt update
+RUN apt install -y ros-noetic-joy ros-noetic-octomap-ros ros-noetic-mavlink
+RUN apt install -y ros-noetic-octomap-mapping ros-noetic-control-toolbox
+RUN apt install -y python3-vcstool python3-catkin-tools protobuf-compiler libgoogle-glog-dev
+# RUN rosdep init
+RUN rosdep update
+RUN cd /catkin_ws/src 
+# RUN catkin_init_workspace
+RUN cd /catkin_ws && rosdep install --from-paths src -y -i
+RUN rosdep update
+RUN catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release -DCATKIN_ENABLE_TESTING=False
+
 RUN cd /catkin_ws && rosdep install --from-paths src --ignore-src -y --rosdistro $ROS_DISTRO
 RUN cd /catkin_ws && rosdep update && rosdep install --from-paths . --ignore-src --rosdistro $ROS_DISTRO -y
-RUN /bin/bash -c '. /opt/ros/$ROS_DISTRO/setup.bash; cd /catkin_ws; catkin build'
+RUN /bin/bash -c '. /opt/ros/$ROS_DISTRO/setup.bash; cd /catkin_ws; catkin build -j1'
+RUN ["cp", "/catkin_ws/build/rotors_gazebo_plugins/Actuators.pb.cc", "/catkin_ws/src/CrazyS/rotors_gazebo_plugins/src/"] 
```

Comment:
use COPY instead, and try to reduce image layers

Reply:
COPY copies the files from local machine to docker container and these files source as well destination is in container itself so 'RUN cp ...' is used here
Layers has been reduced  in the cleaned Dockerfile